methods/account/getAccountTransactions.md

```diff
@@ -260,9 +260,85 @@ The above command returns JSON structured like this:
   }
 ]
 ```
+### Example 4 of 6: Get transactions by number
 
+{% sample lang="bash" %}
+
+```bash
+curl -v -X GET  https://api.catapult.inetwork.com/v1/users/{user-id}/account/transations?number=+19199230192 \
+  -u {{apiToken}}:{secret} \
+  -H "Content-type: application/json" \
+```
+{% sample lang="js" %}
+
+```js
+//Get transactions limiting result
+//Promise
+var params = {
+  number: '+17208352955'
+}
+client.Account.getTransactions(params)
+.then(function (response) {
+    console.log(response.transactions);
+    if(response.hasNextPage) {
+        return response.getNextPage();
+    }
+    else {
+        return {transactions: []};
+    }
+})
+.then(function(response) {
+    console.log(response.transactions);
+});
+```
+
+{% sample lang="csharp" %}
+
+```csharp
+var transactions = await client.Account.GetTransactions(
+  new AccountTransactionQuery {
+    number = +13035659555
+  }
+);
+```
+
+{% sample lang="ruby" %}
+
+```ruby
+transactions = Account.get_transactions(client, {:number => "+19192130923"})
+```
+
+{% common %}
+The above command returns JSON structured like this:
+
```

Comment:
Set the number that you search for the same in each example `+19191231234` and then set the `{number}` in the response below to match that (since we're searching my number, it's good to show that consistency in the response)